sirepo/job_driver/__init__.py

```diff
@@ -53,7 +53,9 @@ def assign_instance_op(req, jobRunMode, op):
         )
     op.driver = res
     op.driver.ops[op.opId] = op
-    op.op_slot = None
+    op.cpu_slot = op.driver.cpu_slot_q.sr_slot_proxy(op)
```

Comment:
I don't think this will work for docker. cpu_slot_q is a dict in the docker case.

Reply:
`DockerDriver.cpu_slot_q` is `SlotQueue`, which [is initialized from `host.cpu_slot_q`](https://github.com/radiasoft/sirepo/blob/01352458a3d7fd84e9f552d7b9ac45e864e2d586/sirepo/job_driver/docker.py#L53).